### PR TITLE
redirect link fixes + HID issue

### PR DESCRIPTION
**Pull request by namkedia**

### CODE REVIEW

File: en-US/Docs/ReleaseNotesRTM.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ title: Release Notes
 permalink: /en-US/Docs/ReleaseNotesRTM.htm
 lang: en-US
 ---
-{% include redirect.html url="https://docs.microsoft.com/en-us/windows/iot-core/release-notes/commercial/creatorsupdate" %}
```
Comment (rachitb777) [inline en-US/Docs/ReleaseNotesRTM.md:7]: This looks wrong

File: Resources/contribute/template/docs-template.md [hunk 2]
```diff
@@ -1,58 +0,0 @@
----
-layout: docs
-title: <title>
```
Comment (rachitb777) [inline Resources/contribute/template/docs-template.md:3]: Why are you deleting this file ? We also have samples on web-sites apart from docs. Please re-write this or leave it be.

File: en-US/Docs/ReleaseNotesRTM.md (hunk 1 above)
Comment (saraclay) [reply en-US/Docs/ReleaseNotesRTM.md]: It should be this: https://docs.microsoft.com/en-us/windows/iot-core/release-notes/commercial/

**Review by saraclay — APPROVED** [review]

File: Resources/contribute/template/docs-template.md (hunk 2 above)
Comment (namkedia) [reply Resources/contribute/template/docs-template.md:3]: We have samples template for the same. Deleting this to ensure that people do not commit docs to WOD.

File: en-US/Docs/ReleaseNotesRTM.md (hunk 1 above)
Comment (namkedia) [reply en-US/Docs/ReleaseNotesRTM.md:7]: This is how Sara wanted the document to look on docs.microsoft.

Comment (saraclay) [reply en-US/Docs/ReleaseNotesRTM.md:7]: We'll want it to land on https://docs.microsoft.com/en-us/windows/iot-core/release-notes/commercial/fallcreatorsupdate once the PR in docs is merged.

**Review by saraclay — APPROVED** [review]

**Pull request closed without merging**